cypress/integration/Header_spec.js

```diff
@@ -0,0 +1,8 @@
+describe('Header shows at the top of the home page', () => {
```

Comment:
Nice work writing the test for this new component, so that when we merge it, we know that what we want to appear on the page is what is appearing.